src/utils/proto-methods.js

```diff
@@ -0,0 +1,7 @@
+// -------------------------------------------------------------
+// WARNING: this file is used by both the client and the server.
+// Do not use any browser or node-specific API!
+// -------------------------------------------------------------
+// GH-245
```

Comment:
It doesn't follows our comment conventions. First of all it require `NOTE: ` prefix. Also, I've asked for the textual description with the reference to the issue.